src/resources.js

```
@@ -1172,64 +1259,73 @@ function loadSpecialResource(name,color) {
+                            break;
+        
+                        case 'evil':
+                            desc.append($(`<span>${loc(`resource_${name}_desc_e`,[+((darkEffect('evil') - 1) * 100).toFixed(2),+((darkEffect('evil',true) - 1) * 100).toFixed(2)])}</span>`));
+                            break;
+        
+                        case 'micro':
+                            desc.append($(`<span>${loc(`resource_${name}_desc_m`,[darkEffect('micro',false),darkEffect('micro',true)])}</span>`));
+                            break;
+        
+                        case 'heavy':
+                            let hDE = darkEffect('heavy');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)